plugin/src/main/java/io/jenkins/plugins/casc/ConfigurationAsCode.java

```diff
@@ -406,17 +405,18 @@ public void doExport(StaplerRequest req, StaplerResponse res) throws Exception {
      * @throws Exception
      */
     @RequirePOST
-    public void doExportNewJSONSchema(StaplerRequest req, StaplerResponse res) throws Exception {
+    public void doNewSchema(StaplerRequest req, StaplerResponse res) throws Exception {
```

Comment:
@oleg-nenashev The code seems to be exact same copy of the `viewExport` but still doesnt show up as a url.

Reply:
I would probably use v2 or beta, new gets outdated really easily

Reply:
Done